ldap/src/main/java/org/springframework/security/ldap/server/UnboundIdContainer.java

```diff
@@ -54,9 +54,31 @@
 
 	public UnboundIdContainer(String defaultPartitionSuffix, String ldif) {
 		this.defaultPartitionSuffix = defaultPartitionSuffix;
+		setLdif(ldif);
+	}
+
+	public String getLdif() {
+		return ldif;
+	}
+
+	public void setLdif(String ldif) {
+		if (!StringUtils.hasText(ldif)) {
+			throw new IllegalArgumentException("ldif file value is missing");
+		}
+		checkFilePath(ldif);
 		this.ldif = ldif;
 	}
 
+	private void checkFilePath(String ldif) {
```

Comment:
This logic should go in the `importLdif` method. That way, it is called on `start()`, rather than in the constructor.

Reply:
do you think it will be also wise to add the validation over that method ?